File: 2015/02/one.py
lines = open("input.txt").read().split("\n")
print(sum([min(b) + 2*sum(b) for b in [[b[0] * b[1],b[1] * b[2],b[2] * b[0]] for b in [[int(x) for x in s.split("x")] for s in lines]]]))

File: 2016/02/one.go
package main

import (
    "fmt"
    "os"
    "strings"
)

func main() {
    x, y := 1, 1

    data, err := os.ReadFile("input.txt")
    if err != nil { panic(err) }

    input := string(data)
    lines := strings.Split(strings.Trim(input, "\n"), "\n")

	code := 0

    for _, line := range lines {
		for _, char := range line {
			if char == 'U' && y > 0 { y-- }
			if char == 'D' && y < 2 { y++ }
			if char == 'L' && x > 0 { x-- }
			if char == 'R' && x < 2 { x++ }
		}

		code *= 10
		code += 1 + x + 3 * y
    }
    
    fmt.Println(code)
}